### PR TITLE
Auto upload ocp baseline daily as well

### PR DESCRIPTION
**Pull request by rullzer**
Signed-off-by: Roeland Jago Douma <roeland@famdouma.nl>

### CODE REVIEW

File: .github/workflows/update-psalm-baseline.yml [hunk 1]
```diff
@@ -44,3 +44,43 @@ jobs:
...
+      - name: Psalm
+        run: composer run psalm -- -c psalm-ocp.xml --monochrome --no-progress --output-format=text --update-baseline
+      - name: Reset composer.json and composer.lock
+        run: |
+          rm -rf lib/composer
+          git checkout -- composer.json composer.lock lib/composer
+      - name: Create Pull Request
+        uses: peter-evans/create-pull-request@v3
+        with:
+          token: ${{ secrets.MACHINE_USER_PAT }}
```
Comment (ChristophWurst) [inline .github/workflows/update-psalm-baseline.yml:74]: @tobiasKaminsky :eyes:

Comment (tobiasKaminsky) [reply .github/workflows/update-psalm-baseline.yml:74]: Please test if this is possible with GITHUB_TOKEN.

Comment (ChristophWurst) [reply .github/workflows/update-psalm-baseline.yml:74]: I thought it was just available for a workflow triggered by a user, but https://docs.github.com/en/actions/reference/authentication-in-a-workflow#about-the-github_token-secret doesn't list any limitation.

Let's give this a try.

Comment (tobiasKaminsky) [reply .github/workflows/update-psalm-baseline.yml:74]: @MorrisJobke what do you think?

**Pull request closed without merging**